src/analysis/normed_space/add_torsor_bases.lean

```diff
@@ -81,77 +81,77 @@ open affine_map
 
 /-- Given a set `s` of affine-independent points belonging to an open set `u`, we may extend `s` to
 an affine basis, all of whose elements belong to `u`. -/
-lemma exists_subset_affine_independent_span_eq_top_of_open {s u : set P} (hu : is_open u)
+lemma is_open.exists_between_affine_independent_span_eq_top {s u : set P} (hu : is_open u)
   (hsu : s ⊆ u) (hne : s.nonempty) (h : affine_independent ℝ (coe : s → P)) :
   ∃ t : set P, s ⊆ t ∧ t ⊆ u ∧ affine_independent ℝ (coe : t → P) ∧ affine_span ℝ t = ⊤ :=
 begin
   obtain ⟨q, hq⟩ := hne,
-  obtain ⟨ε, hε, hεu⟩ := metric.is_open_iff.mp hu q (hsu hq),
+  rcases metric.nhds_basis_closed_ball.mem_iff.1 (hu.mem_nhds $ hsu hq) with ⟨ε, ε0, hεu⟩,
```

Comment:
It might be nice to preserve that this was an `obtain`.

Reply:
Done.